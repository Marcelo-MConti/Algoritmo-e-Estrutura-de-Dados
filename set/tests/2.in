1
20 30
100 96 94 83 79 76 72 69 65 58 49 44 40 35 32 21 14 12 11 10
56 59 66 55 65 84 85 78 61 86 79 53 90 89 64 82 69 54 51 75 81 50 52 72 70 80 74 58 77 71
3
