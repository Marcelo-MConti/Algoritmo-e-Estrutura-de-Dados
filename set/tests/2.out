58, 65, 69, 72, 79,
